From 28281106d11317d2127133234d27ccfa015c4f85 Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Mon, 25 Mar 2013 11:35:57 +0900
Subject: [PATCH 14/15] ARM: shmobile: bockw: fix R-GP2D interrupt number

This patch corrects the R-GP2D interrupt number on Bock-W
and replaces gic_spi() with gic_iid() macro because it can be specified
as an ICCIAR value described in the datasheet, the spi numbers not being listed.
---
 arch/arm/mach-shmobile/board-bockw.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-shmobile/board-bockw.c b/arch/arm/mach-shmobile/board-bockw.c
index c5311db..74fa23b 100644
--- a/arch/arm/mach-shmobile/board-bockw.c
+++ b/arch/arm/mach-shmobile/board-bockw.c
@@ -126,7 +126,7 @@ static struct resource rcar_gp2d_resources[] = {
 		.flags  = IORESOURCE_MEM,
 	},
 	[1] = {
-		.start  = gic_spi(68),
+		.start  = gic_iid(0x5C),
 		.flags  = IORESOURCE_IRQ,
 	},
 };
-- 
1.7.5.4

